src/components/tabs/__internal__/tabs-header/tabs-header.component.tsx

```diff
@@ -1,15 +1,26 @@
-import React from "react";
+import React, { useRef, useState } from "react";
 import {
   StyledTabsHeaderWrapper,
   StyledTabsHeaderList,
+  StyledTabsBottomBorder,
 } from "./tabs-header.style";
+import useThrottle from "../../../../hooks/__internal__/useThrottle";
+
+const fullOpacityThreshold = 128;
```

Comment:
I confess I don't really understand what this is doing and in particular why the  value is 128? `getOpacityValue` is calculating its input as a propertion of 128, but that input is actually the `scrollLeft` property of the element - which is a value in pixels. What's special about 128 pixels, and what are we actually trying to achieve here anyway?

Reply:
One of the requirements of this ticket is to add shadows on the sides of the TabHeader when it is scrollable. There is a prototype of this linked in [this issue](https://github.com/Sage/design-system/issues/7). In this prototype, these shadows, particularly the way that they disappear, is handled by scrolling in another gradient with the color of the background of the Tabs. While this works in prototype, it won't work when implemented in a reusable component because Tabs might be used on various background colors. (Check this example https://css-tricks.com/books/greatest-css-tricks/scroll-shadows/ - there are linear-gradients acting as a `shadow covers` and radial-gradients acting as the shadows themselves). 

Considering all above I had to come up with different solution, so I decided to tie the opacity of the radial-gradient shadow with the current scroll position. Tbh, the magic `128` number is kinda random, I came up with it trying to replicate the way that the shadow disappears in the prototype. I didn't run it with UX yet but this PR is meant to be reviewed by UX anyway.

Reply:
Thanks for the explanation! I'll let this pass and see what UX say about it - hopefully Chromatic will catch any future regressions on this.